packages/e2e-test-utils-playwright/tsconfig.json

```diff
@@ -4,6 +4,8 @@
 		"incremental": false,
 		"composite": false,
 		"module": "CommonJS",
+		"moduleResolution": "node16",
```

Comment:
Ensures dynamic imports are left untouched by TypeScript, otherwise it would transform them into `require()` calls.

Dynamic import can be used to import ESM files in CJS environments.

Reply:
Maybe this should be the default now in `tsconfig.base.json`?